tests/plugins/tableselection/tableselection.js

```diff
@@ -145,6 +145,42 @@
 			mockMouseSelection( editor, [ cells.getItem( 1 ), cells.getItem( 3 ), cells.getItem( 8 ) ], function() {
 				assert.pass();
 			} );
+		},
+
+		// (#2003)
+		'test': function( editor, bot ) {
+			if ( !CKEDITOR.env.gecko ) {
+				assert.ignore();
+			}
+			bot.setData( CKEDITOR.document.getById( 'emptyParagraph' ).getHtml(), function() {
+				var cells = editor.editable().find( 'td' ).toArray(),
+					ranges = CKEDITOR.tools.array.map( cells, function( cell ) {
```

Comment:
We have dedicated function to get ranges for cells in tests: https://github.com/ckeditor/ckeditor-dev/blob/master/tests/plugins/tableselection/_helpers/tableselection.js#L7-L25